### DISCUSSION TITLE
Explore new approaches to help users refactor Jupyter notebooks into Kedro projects, for example GenAI

### DISCUSSION THREAD

**Discussion by astrojuanlu** [kedro-org/kedro#3787] [Idea]
[TBC]

## Evidence markers

- Using Kedro with Jupyter notebooks is the 6th most viewed page in our documentation. 
- DataEngineerOne has many views (5.6k) of [Using Jupyter Notebook with Kedro - Writing Data Pipelines with Kedro 3](https://www.youtube.com/watch?v=3m8r1hh7ZB4)
- Some conference talks about refactoring Jupyter notebook have unfortunately been rejected

**Comment by astrojuanlu** [comment]
Prior art: https://youtu.be/tA5olYLjHPA (PyData Berlin earlier this year, 1700+ attendees in total)

**Comment by astrojuanlu** [comment]
https://blog.jupyter.org/generative-ai-in-jupyter-3f7174824862

**Comment by rashidakanchwala** [comment]
I was thinking should we close this; as we did this using Kedro-MCP, and also Kedro-Pipeline-Builder helps here.? @merelcht , @astrojuanlu
